### PR TITLE
10 - Fix asset exchange rate

### PR DESCRIPTION
**Pull request by layko-andrey**
Changed exchange rates API URL to Binance API

### CODE REVIEW

File: frontend/src/actions/deploy-contract.js [hunk 1]
```diff
@@ -96,12 +96,12 @@ export const convertCurrencies = conversionValues => async dispatch => {
...
   const toCurrency = conversionValues.toCurrency ? conversionValues.toCurrency : 'eth';
-  const apiRequestUrl = `https://openexchangerates.org/api/convert/${fromAmount}/${fromCurrency}/${toCurrency}?app_id=d34199d67d85445a846040c0cf621510`;
+  const apiRequestUrl = `https://api.binance.com/api/v3/avgPrice?symbol=${fromCurrency}${toCurrency}`;
 
   try {
     const data = await dispatch(apiGet(apiRequestUrl));
-    dispatch({ type: CONVERT_CURRENCIES, payload: data.response });
-    return data.response;
+    dispatch({ type: CONVERT_CURRENCIES, payload: data.price *fromAmount });
+    return data.price *fromAmount;
```
Comment (trenatov) [inline frontend/src/actions/deploy-contract.js]: space after * ?

**Review by trenatov — CHANGES_REQUESTED** [review]

File: frontend/src/actions/deploy-contract.js (hunk 1 above)
Comment (ruslanys) [reply frontend/src/actions/deploy-contract.js]: Yes, space is missing.

**Review by ruslanys — COMMENTED** [review]
By the way, did we test locally how it works?

**Review by ruslanys — CHANGES_REQUESTED** [review]
Need to resolve comments

**Pull request merged**